bpf/kmesh/workload/include/backend.h

```diff
@@ -30,15 +30,15 @@ static inline backend_value *map_lookup_backend(const backend_key *key)
     return kmesh_map_lookup_elem(&map_of_backend, key);
 }
 
-static inline int backend_manager(ctx_buff_t *ctx, backend_value *backend_v)
+static inline int waypoint_manager(ctx_buff_t *ctx, __u32 addr, __u32 port)
 {
     int ret;
     address_t target_addr;
     __u32 *sk = (__u32 *)ctx->sk;
     struct bpf_sock_tuple value_tuple = {0};
 
-    if (backend_v->waypoint_addr != 0 && backend_v->waypoint_port != 0) {
-        BPF_LOG(DEBUG, BACKEND, "find waypoint addr=[%u:%u]\n", backend_v->waypoint_addr, backend_v->waypoint_port);
+    if (addr != 0 && port != 0) {
```

Comment:
I would suggest checking this before calling waypoint_manager, it would be more readable

Reply:
fixed